optionB/src/components/corner/view.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Circle } from 'react-konva';
+
```

Comment:
Code comment explaining how we are using `<Circle>` which is to render a `<corner>` from the model. I think its useful to add comment. They can use it for any other purpose too.

Reply:
The rest our components have a 1:1 mapping to `xml` entity names. `Circle` is the only one without this